Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -149,6 +149,14 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
     private fun FragmentRecoveryCertificateDetailsBinding.onNavEvent(event: RecoveryCertificateDetailsNavigation) {
         when (event) {
             RecoveryCertificateDetailsNavigation.Back -> popBackStack()
+            RecoveryCertificateDetailsNavigation.ReturnToPersonDetailsAfterRecycling -> {
```

Comment:
That's acutally the responsibility of the PersonDetailsViewModel..

Reply:
Yes, it was, but due to the way we calculate the blue background to always end at half the height of the first card from the `PersonDetailsFragment`, this bug started to show up. I wasn't able fix it without giving up on that new logic that was introduced by @schauersbergern in #5065. I ended up implementing this solution where we travel straight to the `PersonOverviewFragment` because that's where we want to end up anyway.